public/docs/js/latest/glossary.jade

```diff
@@ -1,35 +1,12 @@
-include _util-fns
-// From ts/glossary.jade, the folder ts/latest/_fragments is generated which contains a bunch of partial files.
-// These partials comprise the glossary,a subset of these partials should be used to generate the glossary for
-// __javascript__ under BASICS.
-!=partial('../../ts/latest/_fragments/glossary-intro')
-//!=partial('../../ts/latest/_fragments/glossary-a-1') not needed in javascript
-!=partial('../../ts/latest/_fragments/glossary-a-2')
-!=partial('../../ts/latest/_fragments/glossary-b-c')
-!=partial('../../ts/latest/_fragments/glossary-d1')
-!=partial('../../ts/latest/_fragments/glossary-d2')
-!=partial('../../ts/latest/_fragments/glossary-e1')
-!=partial('../../ts/latest/_fragments/glossary-e2')
-!=partial('../../ts/latest/_fragments/glossary-f-l')
-!=partial('../../ts/latest/_fragments/glossary-m1')
-!=partial('../../ts/latest/_fragments/glossary-m2')
-!=partial('../../ts/latest/_fragments/glossary-n-s')
-!=partial('../../ts/latest/_fragments/glossary-t1')
-!=partial('../../ts/latest/_fragments/glossary-t2')
-!=partial('../../ts/latest/_fragments/glossary-u-z')
+extends ../../ts/latest/glossary.jade
 
-// NOTE: (ericjim): I am almost certain these lines are doing nothing,
-// so instead I use `!=partial` to include the glossary fragments.
-//+includeShared('{ts}', 'intro')
-//+includeShared('{ts}', 'a2')
-//+includeShared('{ts}', 'b-c')
-//+includeShared('{ts}', 'd1')
-//+includeShared('{ts}', 'd2')
-//+includeShared('{ts}', 'e1')
-//+includeShared('{ts}', 'e2')
-//+includeShared('{ts}', 'f-l')
-//+includeShared('{ts}', 'm1')
-//+includeShared('{ts}', 'n-s')
-//+includeShared('{ts}', 't1')
-//+includeShared('{ts}', 't2')
-//+includeShared('{ts}', 'u-z')
\ No newline at end of file
+block includes
+  include _util-fns
+
+block bootstrap-defn-top
+  :marked
+    We launch an Angular application by "bootstrapping" it with the
+    `bootstrap` method.  The `bootstrap` method identifies an
+    application's top level "root" [Component](#component) and optionally
+    registers service [providers](#provider) with the [dependency injection
+    system](#dependency-injection).
```

Comment:
For some reason I was under the impression that the old definition was appearing in the JS glossary, but you are correct, it should not. I'll remove it and rebase.